doc/USER_GUIDE.md

```diff
@@ -34,45 +34,45 @@ Users have the following options for specifying their own values:
 | Variable | Default value | Description |
 | - | - | - |
 | `additional_yum_repos` | `""` |  |
-| `ami_component_description` | ```{{user `remote_folder`}}/worker``` |  |
-| `ami_description` | ```{{user `remote_folder`}}/worker``` |  |
-| `ami_name` | None |  |
+| `ami_component_description` | ```(k8s: {{ user `kubernetes_version` }}, docker: {{ user `docker_version` }}, containerd: {{ user `containerd_version` }})``` |  |
+| `ami_description` | ```EKS Kubernetes Worker AMI with AmazonLinux2 image``` |  |
+| `ami_name` | *None* |  |
 | `ami_regions` | `""` |  |
 | `ami_users` | `""` |  |
-| `arch` | None |  |
+| `arch` | *None* |  |
 | `associate_public_ip_address` | `""` |  |
-| `aws_access_key_id` | ```{{user `remote_folder`}}/worker``` |  |
-| `aws_region` | ```{{user `remote_folder`}}/worker``` |  |
-| `aws_secret_access_key` | ```{{user `remote_folder`}}/worker``` |  |
-| `aws_session_token` | ```{{user `remote_folder`}}/worker``` |  |
-| `binary_bucket_name` | ```{{user `remote_folder`}}/worker``` |  |
-| `binary_bucket_region` | ```{{user `remote_folder`}}/worker``` |  |
-| `cache_container_images` | ```{{user `remote_folder`}}/worker``` |  |
-| `cni_plugin_version` | ```{{user `remote_folder`}}/worker``` |  |
-| `containerd_version` | ```{{user `remote_folder`}}/worker``` |  |
-| `creator` | ```{{user `remote_folder`}}/worker``` |  |
-| `docker_version` | ```{{user `remote_folder`}}/worker``` |  |
-| `encrypted` | ```{{user `remote_folder`}}/worker``` |  |
-| `instance_type` | None |  |
+| `aws_access_key_id` | ```{{env `AWS_ACCESS_KEY_ID`}}``` |  |
+| `aws_region` | ```us-west-2``` |  |
+| `aws_secret_access_key` | ```{{env `AWS_SECRET_ACCESS_KEY`}}``` |  |
+| `aws_session_token` | ```{{env `AWS_SESSION_TOKEN`}}``` |  |
+| `binary_bucket_name` | ```amazon-eks``` |  |
+| `binary_bucket_region` | ```us-west-2``` |  |
+| `cache_container_images` | ```false``` |  |
+| `cni_plugin_version` | ```v1.2.0``` |  |
+| `containerd_version` | ```1.6.*``` |  |
+| `creator` | ```{{env `USER`}}``` |  |
+| `docker_version` | ```20.10.23-1.amzn2.0.1``` |  |
+| `encrypted` | ```false``` |  |
+| `instance_type` | *None* |  |
 | `kernel_version` | `""` |  |
 | `kms_key_id` | `""` |  |
-| `kubernetes_build_date` | None |  |
-| `kubernetes_version` | None |  |
-| `launch_block_device_mappings_volume_size` | ```{{user `remote_folder`}}/worker``` |  |
-| `pause_container_version` | ```{{user `remote_folder`}}/worker``` |  |
-| `pull_cni_from_github` | ```{{user `remote_folder`}}/worker``` |  |
-| `remote_folder` | ```{{user `remote_folder`}}/worker``` | Directory path for shell provisioner scripts on the builder instance |
-| `runc_version` | ```{{user `remote_folder`}}/worker``` |  |
+| `kubernetes_build_date` | *None* |  |
+| `kubernetes_version` | *None* |  |
+| `launch_block_device_mappings_volume_size` | ```4``` |  |
+| `pause_container_version` | ```3.5``` |  |
```

Comment:
do we need to hardcode this?

Reply:
the pause container version? we could probably use `latest` or something. This documentation is generated from the values in the JSON variable file.